.github/workflows/docker.yml

```diff
@@ -16,6 +16,9 @@ jobs:
     build:
         runs-on: ubuntu-latest
 
+        env:
+            NEXT_PUBLIC_API_URL: ${{ vars.NEXT_PUBLIC_API_URL }}
```

Comment:
Excuse my ignorance, but where are `vars` defined?

Reply:
They're defined in https://github.com/doubleopen-project/dos/settings/variables/actions. At least VSCode also provides autocomplete and warnings in the IDE if they're not defined.

Reply:
> At least VSCode also provides autocomplete and warnings in the IDE if they're not defined.

Uh, that means VSCode does GitHub API calls for that?

Reply:
> > At least VSCode also provides autocomplete and warnings in the IDE if they're not defined.
> 
> Uh, that means VSCode does GitHub API calls for that?

I haven't gone under the hood, but I'm sure [the plugin responsible for that](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions) calls the API.